packages/tools/pluggable-widgets-tools/configs/eslint.ts.base.json

```diff
@@ -17,7 +17,6 @@
         "plugin:react/recommended",
         "plugin:@typescript-eslint/recommended",
         "prettier",
-        "prettier/@typescript-eslint",
```

Comment:
Doesn't feel like this removal belongs here, can you elaborate?

Reply:
It was removed because the new version doesnt require anymore (after eslint-plugin-config v8)